plugins/chunter/src/index.ts

```diff
@@ -62,6 +62,13 @@ export interface ChunterMessage extends AttachedDoc {
   reactions?: number
 }
 
+/**
+ * @public
+ */
+export interface ChunterMessageExtension extends ChunterMessage {
+  mixinClass: Ref<Class<Doc>>
```

Comment:
Will it be simpler to put `presenter` here directly instead of a class that is then used to get the presenter?

Reply:
Presenters should be part of the model, we should not store ref to class / presenter in every object.

Reply:
If we store presenter here, in case we want to change the presenter we would need to migrate all existing objects in the database.